src/Module.ts

```diff
@@ -157,7 +157,9 @@ function handleMissingExport(
 	return importingModule.error(
 		{
 			code: 'MISSING_EXPORT',
-			message: `'${exportName}' is not exported by ${relativeId(importedModule)}`,
+			message: `'${exportName}' is not exported by ${relativeId(
+				importedModule
+			)}, importing module: ${importingModule.id}`,
```

Comment:
You should also use `relativeId` for the importing module to avoid long unreadable paths. `relativeId` prints the path relative to the current working directory and is used for all errors.